security/pkg/server/ca/server.go

```diff
@@ -238,9 +240,21 @@ func NewWithGRPC(grpc *grpc.Server, ca CertificateAuthority, ttl time.Duration,
 	authenticators := []authenticator{&authenticate.ClientCertAuthenticator{}}
 	serverCaLog.Info("added client certificate authenticator")
 
+	rc, err := rest.InClusterConfig()
+	if err != nil {
```

Comment:
If this change creates a client which talks to K8S API server, please move this change and K8S API calls into security/pkg/k8s/.

Reply:
This change doesn't create a new client, it just modifies how we configure `NewKubeJWTAuthenticator`